cmd/kubeadm/app/cmd/alpha/certs.go

```diff
@@ -173,11 +205,54 @@ func generateRenewalFunction(cert *certsphase.KubeadmCert, caCert *certsphase.Ku
 	}
 }
 
-func generateRenewalCommand(cert *certsphase.KubeadmCert, cfg *renewConfig) *cobra.Command {
+func generateEmbeddedCertRenewalFunction(kdir, k string, cfg *renewConfig) func() {
```

Comment:
this return of a naked function is a bit odd vs. wrapping at the variable declaration.

Reply:
fixed